client/js/bluff.js

```diff
@@ -1 +1,70 @@
 // init
+/*
+ suits, values(ranks) declared
+ */
+const suits = ['spades', 'clubs', 'hearts', 'diams']    //Array of Card Suits
+const values = ['A', '2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K']   //Array of possible card-ranks ie. values.
+
+/*
+ * Card and Deck classes to create Card and Deck objects, Card object is used inside Deck.
+ */
+
+class Card {
+  constructor (cardSuit, cardValue) {
+    this.suit = cardSuit    
+    this.value = cardValue
+  }
+}
+
+class Deck {
+  constructor () {
+    this.deck = []
+  }
+
+  formDeck (suits, values) {
+    for (let i = 0; i < suits.length; i++) {
+      for (let j = 0; j < values.length; j++) {

```

Comment:
Hey! A small suggestion: 
If you want to loop through the elements of an array, you can use `array.ForEach` instead of an explicit for loop. Increased readability imo.